.github/workflows/ui-e2e-tests.yml

```diff
@@ -129,6 +129,15 @@ jobs:
         uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020  # v4.4.0
         with:
           node-version: 21
+      - name: "Compile UI assets (for image build fallback)"
```

Comment:
Ah .. Cool, this is only for image falbavck I see - but I guess it should be done before `prepare_breeze` step ?